### PR TITLE
Fixed an issue where single changed files did not required release no…

### PR DESCRIPTION
**Pull request by bakatzir**
…tes update

## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/28143

## Description
```
/Users/bkatzir/dev/demisto/demisto-sdk/.tox/py37/bin/python /Users/bkatzir/dev/demisto/demisto-sdk/demisto_sdk/__main__.py update-release-notes -i Packs/CommonScripts/ -u revision
You are using demisto-sdk 1.1.7.dev52, however version 1.2.0 is available.
You should consider upgrading via "pip3 install --upgrade demisto-sdk" command.
Starting to update release notes.
Collecting all committed files
Collecting all local changed files against the content master
Finished updating release notes for CommonScripts.
Next Steps:
 - Please review the created release notes found at Packs/CommonScripts/ReleaseNotes/1_2_41.md and document any changes you made by replacing '%%UPDATE_RN%%'.
 - Commit the new release notes to your branch.
For information regarding proper format of the release notes, please refer to https://xsoar.pan.dev/docs/integrations/changelog

Process finished with exit code 0

```

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: demisto_sdk/commands/update_release_notes/update_rn.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def is_bump_required(self):
 
     def only_docs_changed(self):
         changed_files = self.added_files.union(self.modified_files_in_pack)
-        if (len(changed_files) == 1 and 'README' in changed_files.pop()) or \
+        if (len(changed_files) == 1 and 'README' in changed_files) or \
```
Comment (roysagi) [inline demisto_sdk/commands/update_release_notes/update_rn.py:158]:
```suggestion
        if (len(changed_files) == 1 and 'README' in changed_files[0]) or \
```

shouldn't it be like this?

Comment (bakatzir) [reply demisto_sdk/commands/update_release_notes/update_rn.py]: No, it is a set and not a list.

Comment (bakatzir) [reply demisto_sdk/commands/update_release_notes/update_rn.py]:
```
>>> a = set('a')
>>> a
{'a'}
>>> 'a' in a
True

```

Comment (bakatzir) [reply demisto_sdk/commands/update_release_notes/update_rn.py:158]: But i see your point. Sol is to copy IMO.

**Review by roysagi — APPROVED** [review]

File: demisto_sdk/commands/update_release_notes/update_rn.py [hunk 2]
```diff
@@ -155,7 +155,8 @@ def is_bump_required(self):
 
     def only_docs_changed(self):
         changed_files = self.added_files.union(self.modified_files_in_pack)
-        if (len(changed_files) == 1 and 'README' in changed_files.pop()) or \
+        changed_files_copy = changed_files  # copying as pop will leave the file out of the set
```
Comment (yaakovi) [inline demisto_sdk/commands/update_release_notes/update_rn.py:158]: this is not a copy, this is a reference.
use `copy.deepcopy()`

**Review by yaakovi — APPROVED** [review]

**Comment by bakatzir** [comment]
Coveralls non related calc, force merging.

**Comment by bakatzir** [comment]
also fixes: https://github.com/demisto/etc/issues/28403

**Pull request merged**